disassemblers/ofrak_ghidra/ofrak_ghidra/ghidra_scripts/GetCodeRegions.java

```diff
@@ -82,7 +82,7 @@ class ResultCodeRegion {
         }
 
         String toJson() {
-            return String.format("{\"start\":%d,\"size\":%d,\"name\":\"%s\"}", start, size, name);
+            return String.format("{\"start\":%d,\"size\":%d,\"name\":\"%s\"}", start, size, name.replace("\"", "\\\""));
```

Comment:
For arbitrary strings, you'd also need to at least escape `\`. Because otherwise I could set `name = 'start\\", ::::'` to make this generate malformed JSON. 

That doesn't warrant changing this PR, though, since all of these names are coming directly from Ghidra, and can't contain slashes or quotes anyway (as far as I can tell)!

Reply:
Yeah, I knew that I wasn't really implementing full input sanitization and with backslashes in particular. It does seem unlikely to run into problems with these names though (section names and function names) as it would be highly unusual for these to include anything but alphanumeric characters plus underscores/dashes.